web/src/main/resources/application.properties

```diff
@@ -7,3 +7,5 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL9Dialect
 spring.jpa.generate-ddl=true
 spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.naming.implicit-strategy=org.hibernate.boot.model.naming.ImplicitNamingStrategyComponentPathImpl
```

Comment:
coś z tym jest nie tak - po dodaniu tych linii zmienia się podłoga_case na wszystkorazemcase (np. sprint -> timeworked)

Reply:
To pozostaje przedyskutowac czy to jest dla nas duzy problem, bo alternatywa jest jedynie manualme przemapowanie kazdego pola osobno

Reply:
@jplewa

Reply:
Mi `wszystkorazemcase` nie przeszkadza wybitnie w tym przypadku, a robienie tego ręcznie wydaje się bardziej problematyczne